src/App.tsx

```diff
@@ -1,223 +1,259 @@
-            <label className="todo__status-label">
-              <input
-                data-cy="TodoStatus"
-                type="checkbox"
-                className="todo__status"
-              />
-            </label>
-
-            <span data-cy="TodoTitle" className="todo__title">Redux</span>
-            <button
-              type="button"
-              className="todo__remove"
-              data-cy="TodoDeleteButton"
-            >
-              ×
-            </button>
-
-            <div data-cy="TodoLoader" className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-        </section>
-
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="todosCounter">
-            4 items left
-          </span>
-
-          <nav className="filter" data-cy="Filter">
-            <a
-              data-cy="FilterLinkAll"
-              href="#/"
-              className="filter__link selected"
-            >
-              All
-            </a>
-
-            <a
-              data-cy="FilterLinkActive"
-              href="#/active"
-              className="filter__link"
-            >
-              Active
-            </a>
-            <a
-              data-cy="FilterLinkCompleted"
-              href="#/completed"
-              className="filter__link"
-            >
-              Completed
-            </a>
-          </nav>
-
-          <button
-            data-cy="ClearCompletedButton"
-            type="button"
-            className="todoapp__clear-completed"
-          >
-            Clear completed
-          </button>
-        </footer>
-      </div>
-
-      <div
-        data-cy="ErrorNotification"
-        className="notification is-danger is-light has-text-weight-normal"
-      >
-        <button
-          data-cy="HideErrorButton"
-          type="button"
-          className="delete"
-        />
-
-        Unable to add a todo
-        <br />
-        Unable to delete a todo
-        <br />
-        Unable to update a todo
-      </div>
+      <TodoContent
+        todos={todos}
+        countOfTodos={countOfTodos}
+        countOfLeftTodos={countOfLeftTodos}
+        hasComplited={hasComplited}
+        visibleTodos={filteredTodos}
+        newTodoField={newTodoField}
+        filterTodos={filterTodos}
+        filterStatus={filterStatus}
+        onChangeTitle={onChangeTitle}
+        todoTitle={todoTitle}
+        createNewTodo={createNewTodo}
+        isLoading={isLoading}
+        deleteTodo={deleteTodo}
+        proccessedTodoId={proccessedTodoId}
+        changeCompleteStatus={changeCompleteStatus}
+        deleteAllCompletedTodos={deleteAllCompletedTodos}
+        copleteAllTodos={copleteAllTodos}
+        isEveryTodosComplited={isEveryTodosComplited}
+        changeTodoTitle={changeTodoTitle}
+      />
```

Comment:
there is so many props for TodoContent component
can we optimize it? let's check this props, probably some of them can be moved and be calculated inside the TodoContent component and be passed as an arguments for the needed functions